<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReactionWheelsInfo</class>
 <widget class="QDockWidget" name="ReactionWheelsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>101</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="allowedAreas">
   <set>Qt::RightDockWidgetArea| Qt::LeftDockWidgetArea</set>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Reaction Wheels</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>398</width>
         <height>87</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>

        <item>
         <widget class="QGroupBox" name="LegendGroup">
          <property name="title"/>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="GraphicGroup">
          <property name="title"/>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="VerboseGroupNew">
          <property name="title">
           <string>RW Verbose</string>
          </property>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="PowerPlotGroup">
          <property name="title">
           <string>RW Power Consumption</string>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
         </widget>
        </item>

        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RW Status</string>
   </property>
  </action>
  <action name="actionVerbose_DisplayNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RW Verbose</string>
   </property>
  </action>
  <action name="actionPowerPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RW Power Consumption</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>

  <connection>
   <sender>actionPowerPlot</sender>
   <signal>toggled(bool)</signal>
  <receiver>PowerPlotGroup</receiver>
  <slot>setVisible(bool)</slot>
   <sender>actionPowerPlot</sender>
   <signal>toggled(bool)</signal>
   <receiver>PowerPlotGroup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVerbose_DisplayNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>VerboseGroupNew</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
